docs/ExpectAPI.md

```diff
@@ -680,12 +680,14 @@ describe('grapefruits are healthy', () => {
 });
 ```
 
-### `.toMatchObject(object)`
+### `.toMatchObject(object, ordered)`
```

Comment:
Could it be an `options` object, perhaps? Would be easier to add something more in the future. Named option would also make it easier to understand what it does without any need to look at docs.

What if the option would be called `ignoreElementsOrder` or `ignoreOrder`?

Example:

```ts
// current
expect(receivedObj).toMatchObject(actualObj, false);

// proposed
expect(receivedObj).toMatchObject(actualObj, {ignoreElementsOrder: true});
```

Reply:
Thanks for the good naming advice. I have changed the parameter name and adjusted the types.